### PR TITLE
BaSP- TG-6: Project table - resources list

### PR DESCRIPTION
**Pull request by pabloschivazappa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.1",
```
Comment (leimeter-joaquin) [inline package.json:11]: Remove this change to package.json

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (pabloschivazappa) [reply package.json:11]: Done

**Review by leimeter-joaquin — APPROVED** [review]
There are some issues, but we can re iterate on them in the following week

File: src/Components/Projects/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+            <div>
+              <label className={formStyles.form}> Add Employees</label>
+              {employees.map((employee, index) => (
+                <div key={index} id="employee-form">
+                  <label>Employee</label>
+                  <input
+                    className={formStyles.inputs}
+                    type="text"
+                    name="employee"
+                    value={employees.map((e) => e.employee).map((id) => id._id)}
```
Comment (leimeter-joaquin) [inline src/Components/Projects/Form/index.js:187]:
```suggestion
                    value={employees[index]._id}
```

Check if this works better. I cannot write into the employee id input. 
Also, for next week, we should work on showing the user the available employees and passing a name on the input, not an id.

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

File: src/Components/Projects/Form/index.js (hunk 2 above)
Comment (pabloschivazappa) [reply src/Components/Projects/Form/index.js]: Done with "value={employees[index].employee._id}"

**Review by leimeter-joaquin — APPROVED** [review]

**Review by Alfondalix — DISMISSED** [review]

**Comment by eialyafe** [comment]
All functionalities are ok edit, delete, and create. All success or error messsage are displeyed. 
There were just syntax mistakes:
 
1) All error messages when editting projects says: "cannot CREATE"
 
2) All error messages when create or edit with some employees fields empty says: "Cannot create project: "employees[0].employee"

 3) When you try create a project with start date grater than end date message displayed is: "Cannot create project: "endDate" must be greater than "ref:startDate", just need to be deleted "ref"

I made a doc  with all messages error thas had a problem, and with all testes made. You can check it here 
https://docs.google.com/spreadsheets/d/1q_hpcj_gpgHIVMhLFIMpEQzGRzcBr-4YIwoG4fL3cfo/edit?usp=sharing

**Review by amilcarHernandez24 — DISMISSED** [review]

**Review by amilcarHernandez24 — APPROVED** [review]
Well done.

**Review by leimeter-joaquin — APPROVED** [review]

**Pull request merged**